plugins/ci/pkg/ci/helm.go

```diff
@@ -101,9 +103,16 @@ func handleRemoteHelmChart(helm models.HelmConfig, tempFolder string, configFold
 
 func doHandleRemoteHelmChart(helm models.HelmConfig, chartName, chartVersion string, fluxValues map[string]interface{}, tempFolder, configFolder string) error {
 	repoName := fmt.Sprintf("%s-%s-repo", helm.Name, chartName)
-	_, err := commands.ExecWithMessage(exec.Command("helm", "repo", "add", repoName, helm.Repo), "Adding chart repository: "+repoName)
+	output, err := commands.ExecWithMessage(exec.Command("helm", "repo", "add", repoName, helm.Repo), "Adding chart repository: "+repoName)
 	if err != nil {
-		return err
+		return models.ScanErrorsReportResult{
+			ErrorMessage: fmt.Sprintf("%v: %s", err, output),
+			ErrorContext: fmt.Sprintf("adding helm repository %q", helm.Repo),
+			Kind:         "HelmChart",
+			ResourceName: helm.Name,
+			Filename:     helm.Name,
```

Comment:
Is there something else we should use as the CI filename for scan-errors associated with a Helm chart? Here I use the name of the fairwinds-insights.yaml Helm definition.

Reply:
Maybe having the `path` as filename would be useful?

Reply:
I'll update the error message filename when the Helm chart is local (contained in the repo) to use the chart path. I'll keep remote charts using the Helm chart name from fairwinds-insights.yaml, I can't think of a better filename to use in that case.